recipes-multimedia/pulseaudio/pulseaudio_%.bbappend

```diff
@@ -0,0 +1,6 @@
+COMPATIBLE_MACHINE = "amlogic-s400"
```

Comment:
This will block pulseaudio on other machines !

Use :
```
SRC_URI_append_amlogic-s400 = "file://0001-s400-set-default-sink-source-to-alsa-default.patch"
```